timestamp.go

```diff
@@ -0,0 +1,200 @@
+package pkcs7
+
+// Warn: before merging, check https://github.com/github/smimesign/blob/main/ietf-cms/LICENSE.md
```

Comment:
As I understand, it is completely ok as this repo is also in MIT licence. So we do not change the licence of the code

Reply:
Code as been moved in another project